mixer/src/Meter_Module.C

```diff
@@ -58,7 +59,18 @@ Meter_Module::Meter_Module ( )
     p.connect_to( new float[1] );
     p.control_value_no_callback( 0 );
 
+    // public port for meter level
+    Port p2( this, Port::OUTPUT, Port::CONTROL, "Level (dB)" );
+    p2.hints.type = Port::Hints::LINEAR;
+    p2.hints.ranged = true;
+    p2.hints.maximum = 6.0f;
+    p2.hints.minimum = -70.0f;
+    p2.hints.dimensions = 1;
+    p2.connect_to( new float[1] );
+    p2.control_value_no_callback( 0 );
+
     add_port( p );
+    add_port( p2 );
```

Comment:
Additional port for dB value computed at UI update rate (feedback from the other is not usable).